__tests__/redirects.ts

```diff
@@ -0,0 +1,256 @@
+/**
+ * This file is part of Serlo.org Cloudflare Worker.
+ *
+ * Copyright (c) 2020 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2020 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/serlo.org-cloudflare-worker for the canonical source repository
+ */
+
+import { handleRequest } from '../src'
+import { Instance } from '../src/utils'
+import {
+  expectIsNotFoundResponse,
+  givenApi,
+  givenUuid,
+  mockHttpGet,
+  returnsMalformedJson,
+  returnsText,
+} from './__utils__'
+
+async function handleUrl(url: string): Promise<Response> {
+  return await handleRequest(new Request(url))
+}
+
+function expectToBeRedirectTo(response: Response, url: string, status: number) {
+  expect(response.headers.get('Location')).toBe(url)
+  expect(response.status).toBe(status)
+}
+
+describe('Redirects', () => {
```

Comment:
```suggestion
```
This block can be deleted. (You need to also delete `})` and the end of the block and you need to format this file).